stable/anchore-admission-controller/values.yaml

```diff
@@ -93,6 +93,7 @@ tolerations: []
 ## Constrain which nodes your pod is eligible to be scheduled on, based on labels on the node
 ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
 affinity: {}
+labels: {}
```

Comment:
Should we rename this `extraLabels` to indicate that it's additive to the labels that currently exist in the helper template?

Reply:
Done. thanks!